src/java/api.js

```diff
@@ -0,0 +1,16 @@
+const RealmAPI = require('../index')
+
+module.exports = class JavaRealmAPI extends RealmAPI {
+  async getRealmAddress (realmId) {
+    return await this.rest.get(`/worlds/v1/${realmId}/join/pc`)
```

Comment:
Return values for the API should be explicit

Reply:
Ah yes, added that now